packages/frontend-2/components/workspace/header/Header.vue

```diff
@@ -77,6 +77,7 @@
         v-if="!isWorkspaceGuest"
         :workspace-info="workspaceInfo"
         :is-workspace-admin="isWorkspaceAdmin"
+        :is-workspace-guest="isWorkspaceGuest"
```

Comment:
It's not directly relevant, but I've been thinking about creating a composable that holds this basic WS information, now you will always be in 1 active workspace. Something like `useActiveWorkspace()` so we can just do `activeWorkspace.role` or `activeWorkspace.isAdmin`. wdyt?